PCbuild/tcltk.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="pyproject.props" Condition="$(__PyProject_Props_Imported) != 'true'" />
   <PropertyGroup>
-    <TclVersion Condition="$(TclVersion) == ''">8.6.14.0</TclVersion>
+    <TclVersion Condition="$(TclVersion) == ''">9.0.0.0</TclVersion>
```

Comment:
This should match the version in `get_externals.bat`. It can be overridden with the `TclVersion` environment variable if you're doing a custom build.

Reply:
I don't know what is the best choice for how to control whether _tkinter is built with Tk 8.6.15 or Tk 9.0.0. 

The way that I created externals/tcltk-9.0.0.0 was to build Tcl 9.0 and Tk 9.0 using nmake in the win directories of those two distributions.  The commands needed were:
`nmake -f makefile.vc`
`nmake -f makefile.vc INSTALLDIR=C\path\to\externals\tcltk-9.0.0.0\amd64 install`

I realize that to make this work in its final form tcltk-9.0.0.0 should be provided by get_externals.bat and there should be some sort of switch that controls which version gets used by build.bat.  Finalizing that design is out of the scope of this PR and beyond my expertise.  This PR is meant to be about changes to _tkinter.c and the test scripts which will be needed to make them work with both TclTk 8.6.15 and 9.0.0.

Reply:
Overriding `TclVersion` and `tcltkDir` is the way, but changing the default `TclVersion` as you've done in the PR will break everyone else until we switch fully to relying on 9.0. So I suggest keeping it at 8.6 for now, override it yourself for testing, and when we're ready to switch to 9.0 by default then we can update it (there are more things that have to change before that's possible, so I wouldn't expect it to be in this PR).